Translation/Extractor/File/ValidationExtractor.php

```diff
@@ -86,7 +86,9 @@ public function __construct($metadataFactory)
     public function enterNode(Node $node)
     {
         if ($node instanceof Node\Stmt\Namespace_) {
-            $this->namespace = implode('\\', $node->name->parts);
+            if (isset($node->name)) {
+                $this->namespace = implode('\\', $node->name->parts);
+			}
```

Comment:
Oh boy. This stuff only happens if you use Notepad because you're too lazy to open PHPStorm... Fixed!